docker/bin/stop-presto

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# This works for worker only. Coordinator doesn't support graceful shutdown.
+# This script will block until the server has actually shutdown.
+set -x
+
+http_port="$(cat /usr/lib/presto/etc/config.properties | grep 'http-server.http.port' | sed 's/^.*=\(.*\)$/\1/')"
```

Comment:
isn't `/etc/presto/config.properties` a better path?

Reply:
https://github.com/prestosql/presto/blob/9336f17eb44c81692bd776cdef111f804f420263/docker/bin/run-presto#L5-L11
Base on the current `run-presto` implementation, `/usr/lib/presto/etc` is guarantee to be safe. `/etc/presto` is an optional path for user to mount config into docker.